WordPressKit/WordPressKit/Activity.swift

```diff
@@ -1,62 +1,66 @@
 import Foundation
 
-public struct RemoteActivity {
+public struct Activity {
```

Comment:
Good call on removing the `Remote` prefix, we should consider it for all of `WordPressKit` models